_data/accessibility-tests/in-page-navigation.yml

```
@@ -0,0 +1,91 @@
+title: In-page navigation
+component_status: pass
+test_items:
+  # General tests
+  - summary: In-page navigation adjusts on mobile.
+    summary_additional: When you rotate between portrait mode and landscape mode on a mobile device, in-page navigation is not restricted to a single view.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.4
+  - summary: Color is not the only method used to convey in-page navigation link states.
+    summary_additional: When you view the in-page navigation links, you can read text to understand what page you are on. There may also be underlining missing to indicate you are on the current page.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.1
+  - summary: In-page navigation text and background meet color contrast requirements.
+    summary_additional: When you use a color contrast checker to view the in-page navigation, contrast between the text and background colors is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.3
+  - summary: Navigation links clearly indicate location on the website.
+    summary_additional: When you view the in-page navigation, there is distinct styling that gives clear visual indication of where you are on the page. When using a screen reader, you will hear "current" as the indicator for location.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.8
```

Related to @alex-hull's [comment](https://github.com/uswds/uswds-site/pull/2863#pullrequestreview-2373401993), we've updated this test status

>Criteria 2.4.8 under the general sections should be marked as conditional versus marked as pass. Basing this on previous discussion. Everything else looks good!

```suggestion
  - summary: Navigation links clearly indicate location on the website.
    summary_additional: When you view the in-page navigation, there is distinct styling that gives clear visual indication of where you are on the page. When using a screen reader, you will hear "current" as the indicator for location.
    test_status: conditional
    test_type: general
    version_tested: 3.8.1
    wcag_criterion: 2.4.8
```

---

Resolved in [bad51be](https://github.com/uswds/uswds-site/pull/2863/commits/bad51be98a8c4bb0366197b2eec3e597a1d341b8)